package.json

```diff
@@ -101,6 +100,7 @@
     "karma-sinon": "1.0.5",
     "karma-webpack": "2.0.3",
     "lodash": "3.10.1",
+    "manageiq-api-mock": "https://github.com/ManageIQ/manageiq-api-mock.git#master",
```

Comment:
again, the whole url isn't necessary, referencing the user repo and branch or version is all ya need here

prefer the follow: `ManageIQ/manageiq-api-mock#master`